### PR TITLE
feat(signals): initial setup

### LINKED ISSUE #3992: @ngrx/signals: Add initial library setup
### Information

Add initial setup for the `@ngrx/signals` package.

### I would be willing to submit a PR to fix this issue

- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by markostanimirovic**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

Closes #3992

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: modules/signals/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@ngrx/signals",
+  "version": "0.0.0",
+  "description": "Powerful Extensions for Angular Signals",
```
Comment (markostanimirovic) [inline modules/signals/package.json:4]: let me know if you have a better suggestion for the package description 👀

Comment (brandonroberts) [reply modules/signals/package.json:4]: My recommendation is "Reactive State Management with/for Angular Signals"

**Review by timdeschryver — APPROVED** [review]
Do we also need to exclude signals here @brandonroberts ?
The rest LGTM 🚀


[https://github.com/markostanimirovic/platform/blob/0fcf1371534094f43d92b1077ad5bd21d0e25a95/build/config.ts#L23-L24](https://github.com/markostanimirovic/platform/blob/0fcf1371534094f43d92b1077ad5bd21d0e25a95/build/config.ts#L23-L24)

**Comment by markostanimirovic** [comment]
Good point @timdeschryver! I excluded the signals package from releasing flow since it will be released separately in the beginning.

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**